cirq/circuits/circuit_test.py

```diff
@@ -3817,3 +3817,31 @@ def test_zip():
             cirq.Circuit(cirq.X(a), cirq.CNOT(a, b)),
             cirq.Circuit(cirq.X(b), cirq.Z(b)),
         )
+
+
+def test_repr_html_escaping():
+
+    class TestGate(cirq.Gate):
+
+        def __init__(self, n_qubits, label):
+            self.n_qubits = n_qubits
+            self.label = label
+
+        def num_qubits(self):
+            return self.n_qubits
+
+        def _circuit_diagram_info_(self, args):
+            return cirq.CircuitDiagramInfo(wire_symbols=[self.label] *
+                                           self.n_qubits)
```

Comment:
Keep this a simple as possible for the test.  You can remove the `__init__` method and have the other methods hard-code `2` and `"< ' F ' >"`.

Reply:
Yeah. @cduck please have a look at commit [bac6c2f](https://github.com/quantumlib/Cirq/pull/3221/commits/bac6c2f309a6d9cc9c9ba4ee64354ec6d635486b). Passed the format test in the last commit.